runtime/onert/core/include/exec/Executors.h

```diff
@@ -37,14 +52,16 @@ class Executors
   Executors(Executors &&) = default;
 
   // TODO Use Executor index
-  void emplace(ir::SubgraphIndex idx, std::unique_ptr<IExecutor> exec)
-  {
-    _executors.emplace(idx, std::move(exec));
-  }
+  void emplace(const ir::ModelIndex &model_index, const ir::SubgraphIndex &subg_index,
+               std::unique_ptr<IExecutor> exec);
+
+  // Temporary method for builtin backend migration
+  // TODO Remove this method
+  IExecutor *at(ir::SubgraphIndex idx) const { return at(ir::ModelIndex{0}, idx); }
 
-  IExecutor *at(ir::SubgraphIndex idx) const { return _executors.at(idx).get(); }
+  IExecutor *at(const ir::ModelIndex &model_index, const ir::SubgraphIndex &subg_index) const;
 
-  IExecutor *entryExecutor() const { return at(ir::SubgraphIndex{0}); }
+  IExecutor *entryExecutor() const { return at(ir::ModelIndex{0}, ir::SubgraphIndex{0}); }
```

Comment:
@hseok-oh I didn't fully understand and notice your assumption you wrote in https://github.com/Samsung/ONE/pull/9762#issuecomment-1254543579. At first, I understood {model, subg} of pkg_input[0] would be entryExecutor while you assume that the subgraph {model_idx, subg_idx} = {0, 0} always has pkg input. We may need to update nnpkg specification and add a validation check whether pkg_input contains {0, 0}. I am okay about this assumption.